### PR TITLE
add jupyter

### PR DESCRIPTION
**Pull request by tarquasso**
add jupyter requirements for docker

### CODE REVIEW

**Comment by tarquasso** [comment]
Added a mkdir -p flag (Dave had not committed that change from bronco)
Also, fix key server issue

File: drake-torch.dockerfile [hunk 1]
```diff
@@ -7,6 +7,9 @@ WORKDIR /root
 # https://github.com/phusion/baseimage-docker/issues/58#issuecomment-47995343
 RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
 
+RUN apt-get update && apt-get install -y gnupg2
+
+ENV APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn
```
Comment (duncancmt-dexai) [inline drake-torch.dockerfile]: This seems dangerous. Why is this required?

Comment (tarquasso) [reply drake-torch.dockerfile]: that don't warn line suppresses a warning that otherwise is caused by:
RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 3B4FE6ACC0B21F32
I remove the keyword again... warning is ok to have
